packages/rest-client-jsonapi/src/services/jsonapi/jsonapi.ts

```diff
@@ -1,4 +1,5 @@
 import type { Source, SchemaMap, Resource } from "@hatchifyjs/rest-client"
+import type { Fields, Include, Schemas } from "@hatchifyjs/rest-client"
```

Comment:
Is there a reason why we are doing 2 different imports from the same package?

Reply:
nope, good catch 😄